src/wayland-eglswap.c

```diff
@@ -30,6 +30,8 @@
 #include <assert.h>
 #include <wayland-egl-backend.h>
 
+void wlEglResizeSurfaceIfRequired(WlEglDisplay *display, WlEglPlatformData *pData, WlEglSurface *surface);
```

Comment:
Should this be added to `include/wayland-eglsurface.h`?

Reply:
Yup, I was going to do this, but forgot. Thanks!